src/generator/renderer.clj

```diff
@@ -264,13 +269,28 @@
       [:h2 (get-in contentful-map [:articleList :websiteTitle])]]
      (into [:ul.post-list] (mapv render (get-in contentful-map [:articleList :linkedFrom :postCollection :items])))]))
 
-; Todo implement two person logics: embedded and person page.
+(defmethod render "PeopleList"
+  [args]
+  (let [contentful-map (contentful/get-contentful
+                        :people-by-list-query {:listId (get-in args [:sys :id])})]
+    [:section.people-list-wrap 
+     (into [:ul.people-list] (mapv render (get-in contentful-map [:peopleList :linkedFrom :personCollection :items])))]))
+
+;; Renders People-list and person page
 (defmethod render "Person"
   [args]
-  [:div.person
-   [:span.name (:name args)]])
+  (let [fullbody (empty? (:shortText args))
+        content (if (empty? (:shortText args)) (:description args) (:shortText args))]
+    [(if fullbody :div.person :li.person-body)
+     [:a {:href (create-url (:slug args))} (if fullbody [:h2 (:name args)] [:h3 (:name args)])]
+     [:div.image
+      [:img {:alt (get-in args [:picture :title]) :src (:url (:picture args))}]]
+     [:div.body
+      (into [:div.types] (mapv #(vector :span {:class (str "type " %)} %) (:type args)))
+      [:h4 (:jobTitle args)]
+      [:p.office (:name (:office args))]
```

Comment:
Office is not always present: See person list graphql. This would output an empty paragraph in those cases. Ideally you wouldn't have such an element present if office is not present.